config/routes.rb

```diff
@@ -4,7 +4,7 @@
 
     scope '/api', defaults: {format: :json} do
       post 'facebook/fetch'
-
+      post 'locale/set'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.